tests/alfacase/test_case_description.py

```diff
@@ -788,14 +810,20 @@ def test_pvt_model_table_parameters_description_equal():
     assert table_params_3 != table_params_2
```

Comment:
hmm, that `__eq__` implementation was only there for these test I assume....

Reply:
I did not find any usage of the equality operator outside this test.